gslib/utils/copy_helper.py

```diff
@@ -1272,6 +1278,14 @@ def _DoParallelCompositeUpload(fp,
   return elapsed_time, composed_object
 
 
+def _WarnOnceAboutKmsCompose(logger):
+    with kms_compose_warning_lock:
+        if kms_compose_warning:
+            kms_compose_warning = False
+            logger.warning('Not using parallel composite upload for KMS-encryption. ' +
```

Comment:
Nit: + sign is not required here. The string will be concatenated without it as well.

Reply:
Updated